apps/condo/domains/contact/schema/Contact.js

```diff
@@ -96,13 +96,14 @@ const Contact = new GQLListSchema('Contact', {
     },
     hooks: {
         validateInput: async ({ resolvedData, operation, existingItem, addValidationError, context }) => {
-            const { property, unitName, name, phone, deletedAt } = resolvedData
+            const newItem = { ...existingItem, ...resolvedData }
+            const { property, unitName, name, phone } = newItem
             const [contact] = await ContactAPI.getAll(context, {
                 property: { id: property },
                 unitName,
                 name,
                 phone,
-                deletedAt,
+                deletedAt: null,
```

Comment:
It’s by default here. You can just remove this key

Reply:
Changed to native keystone request, where deletedAt is required

Reply:
It looks like a ADR4 violdtion. You add the `deletedAt` inside the business logic.